File: app/src/main/java/com/shar2wy/moviesapp/models/TrailersResponse.java
package com.shar2wy.moviesapp.models;

import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * Created by shar2wy on 4/8/17.
 */

public class TrailersResponse {
    @SerializedName("results")
    private List<Trailer> results;

    public TrailersResponse() {
    }

    public List<Trailer> getResults() {
        return results;
    }

    public void setResults(List<Trailer> results) {
        this.results = results;
    }
}

File: app/src/main/java/com/shar2wy/moviesapp/models/trailerRepo/TrailersResponse.kt
package com.shar2wy.moviesapp.models.trailerRepo

import com.google.gson.annotations.SerializedName

/**
 * Created by shar2wy on 4/8/17.
 */

class TrailersResponse {
    @SerializedName("results")
    var results: List<Trailer>? = null
}
